packages/components/src/components/hds/super-select/index.hbs

```diff
@@ -0,0 +1,30 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{! NOTE: naming is temporary to not conflict with existing power-select wrapper }}
+<div class={{this.classNames}}>
+  {{!
+    Required:
+    * onChange
+    * options or search
+  }}
+  <PowerSelect
+    @ariaDescribedBy={{@ariaDescribedBy}}
+    @ariaInvalid={{@ariaInvalid}}
+    @ariaLabel={{@ariaLabel}}
+    @ariaLabelledBy={{@ariaLabelledBy}}
+    @disabled={{@disabled}}
+    @onChange={{@onChange}}
+    @options={{@options}}
+    @renderInPlace={{@renderInPlace}}
+    @searchEnabled={{@searchEnabled}}
+    @selected={{@selected}}
+    @verticalPosition={{@verticalPosition}}
+    ...attributes
+    as |option term|
+  >
+    {{yield option term}}
```

Comment:
is `term` meant to be used for a matcher function? if so, it would be good to have an example

Reply:
I'll just remove that for now. It was from an example I was looking at.